src/components/CallView/shared/PresenterOverlay.vue

```diff
@@ -150,11 +150,10 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-/* stylelint-disable csstools/use-logical */
 .presenter-overlay {
 	position: absolute;
 	top: 0;
-	left: 0;
+	inset-inline-start: 0;
```

Comment:
it's breaking upstream library
you can try change start position, checking `isRTL`